orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -293,6 +293,7 @@ def backward(ctx, *grad_output):
                 results = [torch.tensor([1])] * len(self._onnx_graphs_info.user_input_names)
                 results += [_ort_output_to_torch_tensor(backward_output) \
                     for backward_output in backward_outputs[:len(self._onnx_graphs_info.initializer_grad_names_to_train)]]
+                self._gradient_io_binding.clear_binding_outputs()
```

Comment:
Add a comment explaining, the ORTValue is a shared pointer and this call only destroy the IOBinding instance of ORTValue, there is another instance managed by DLTensor.

Reply:
done